packages/actionmenu/src/css/index.js

```diff
@@ -1,6 +1,6 @@
 import core from '@pluralsight/ps-design-system-core'
 
-import * as vars from '../vars'
+import * as vars from '../vars/index.js'
```

Comment:
This might be a bigger convo, not sure.  We can have any time. https://docs.google.com/document/d/1ek9NzuJdQTWBzAwgQwlTQRUdU4ZS-hCg2qQiGU4VSQk/edit

Reply:
interesting. thanks for bring it up

something like https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/extensions.md might help out

Reply:
Yes, that type of eslint plugin might help.  Maybe we leave it on the retro docket for later today.  Since you've approved the PR in total, I take that as ok to move forward with what's committed.  If we have a later convo and change our mind about this generally, I'm happy to change this.